common/src/main/scala/com/avast/cactus/ConverterMethods.scala

```diff
@@ -0,0 +1,44 @@
+package com.avast.cactus
+
+import cats.implicits._
+
+trait ConverterMethods[A, B] {
+  self: Converter[A, B] =>
+
+  final def map[C](f: B => C): Converter[A, C] = new Converter[A, C] {
+    override def apply(fieldPath: String)(a: A): ResultOrErrors[C] = self.apply(fieldPath)(a).map(f)
+  }
+
+  final def flatMap[C](c: Converter[B, C]): Converter[A, C] = new Converter[A, C] {
+    override def apply(fieldPath: String)(a: A): ResultOrErrors[C] = self.apply(fieldPath)(a).flatMap(c.apply(fieldPath))
+  }
+
+  final def flatMap[C](c: B => ResultOrErrors[C]): Converter[A, C] = new Converter[A, C] {
+    override def apply(fieldPath: String)(a: A): ResultOrErrors[C] = self.apply(fieldPath)(a).flatMap(c)
+  }
+
+  final def mapSeq[C](f: Seq[B] => C): Converter[Seq[A], C] = new Converter[Seq[A], C] {
```

Comment:
Why do you need to `map` a sequence? Do you have some real example?

Reply:
I do. Ruda has api which returns a `Seq[A]`. The gRPC service uses `ResponseGpb` as an envelope, while it contains only `repeated ResponseGpbItem` field. The Cactus is able to automatically derive `Converter[A, ResponseGpbItem]` and with `mapSeq` you can easily turn it into `Converter[Seq[A], ResponseGpb]` which seems totally legit to me (I needed this recently too).

Reply:
Well it's definitely handy for the developer but what I don't like about this is that it's a special case, it's not a general principle. It should be a `Converter` from some case class with `Seq[Something]` into `ResponseGpb` right? But you don't want to have the wrapper case class so you just make it possible to avoid it by considering `Seq[Something]` as the wrapper case class, am I right?

Reply:
Exactly. I consider it as common use-case, even though it's _special_.

Reply:
Doesn't common use case and special contradicts each other?  I also vote for keeping the API as simple/minimal as possible